examples/agent/websocket/no_mic/Program.cs

```diff
@@ -0,0 +1,345 @@
+﻿// Copyright 2024 Deepgram .NET SDK contributors. All Rights Reserved.

```

Comment:
@naomi-lgbt  I added this no mic example so it can be used in our Guide, it works like the Node example we have, interacting with the agent, but not invoking a mic in the terminal.